web/html/src/manager/proxy/container-config.tsx

```diff
@@ -235,6 +238,7 @@ export function ProxyConfig() {
           name="sslMode"
           label={t("SSL certificate")}
           title={t("SSL certificate")}
+          hint={"Whether the SSL certificate is going to be created or reuse an existing one"}
```

Comment:
I would suggest a slight rewording, a-la `"Whether to create an SSL certificate or reuse an existing one"`